sqlup-mode.el

```diff
@@ -67,19 +67,12 @@
 this mode's logic will be evaluated.")
 
 (defconst sqlup-eval-keywords
-  '((postgres "EXECUTE"))
-"List of keywords introducing eval strings, organised by dialect.")
+  '((postgres "EXECUTE" "format("))
```

Comment:
The following test passes on your branch:

```cucumber
 Scenario: Upcase a normal SQL keyword preceded only by format
    Given I turn on sqlup-mode
    And I start an action chain
    And I press "M-x"
    And I type "sql-highlight-postgres-keywords"
    And I execute the action chain
    And I insert "format("
    And I insert "'select"
    And I type " "
    Then I should see "format('SELECT "
```

Is this expected behavior? If this is inaccurate, is this acceptable anyway?